client/my-sites/jetpack-manage-error-page/index.jsx

```diff
@@ -21,35 +22,35 @@ module.exports = React.createClass( {
 		optInManage: 'actionCallbackActivate'
 	},
 
-	actionCallbackActivate: function() {
+	actionCallbackActivate() {
 		analytics.ga.recordEvent( 'Jetpack', 'Activate manage', 'Site', this.props.site ? this.props.site.ID : null );
 	},
 
-	actionCallbackUpdate: function() {
+	actionCallbackUpdate() {
 		analytics.ga.recordEvent( 'Jetpack', 'Update jetpack', 'Site', this.props.site ? this.props.site.ID : null );
 	},
 
-	getSettings: function() {
-		var version = this.props.version || '3.4', defaults;
-		defaults = {
+	getSettings() {
+		const version = this.props.version || '3.4';
+		const defaults = {
 			updateJetpack: {
 				title: this.translate( 'Your version of Jetpack is out of date.' ),
 				line: this.translate( 'Jetpack %(version)s or higher is required to see this page.', { args: { version: version } } ),
 				action: this.translate( 'Update Jetpack' ),
 				illustration: null,
-				actionURL: ( this.props.site && this.props.site.jetpack ) ?
-				'../../plugins/jetpack/' + this.props.site.slug :
-					false,
+				actionURL: ( this.props.site && this.props.site.jetpack )
+					? '../../plugins/jetpack/' + this.props.site.slug
+					: false,
 				version: version
 			},
 			optInManage: {
-				line: this.translate( 'Remote management is required to see this page.' ),
-				title: null,
+				title: this.translate( 'Looking to manage this site\'s plugins?' ),
+				line: this.translate( 'We need you to enable the Manage feature in the Jetpack plugin on your remote site' ),
 				illustration: null,
-				action: this.translate( 'Turn on remote management' ),
-				actionURL: ( this.props.site && this.props.site.jetpack ) ?
-					this.props.site.getRemoteManagementURL() :
-					false,
+				action: this.translate( 'Enable Jetpack Manage' ),
+				actionURL: ( this.props.site && this.props.site.jetpack )
+					? this.props.site.getRemoteManagementURL()
```

Comment:
I think we need to append this with `&section=plugins` so that when activating manage we are taken to the right section.